src/main/java/testUI/BaseTest.java

```diff
@@ -0,0 +1,36 @@
+package testUI;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+
+import java.time.Duration;
+
+public class BaseTest {
```

Comment:
Won't we use DriverFactory? That driver and path work ONLY for your local environment. Don't push files like this.